articles/api-auth/tutorials/use-one-rs-for-apis.md

```diff
@@ -0,0 +1,108 @@
+
+* `read:samples`
+* `delete:samples`
+
+For our sample application, we'll add two scopes:
+
+* `read:calendar`;
+* `read:contacts`.
+
+You can think of each one as a microservice.
+
+![](/media/articles/api-auth/tutorials/multiple-apis-one-resource-server/new-scopes.png)
+
+At this point, you've:
+
+* Fully configured your Auth0 API;
+* Added the scopes that correspond to the multiple APIs that the Auth0 API represents.
+
+## Grant Access to the Auth0 API
+
+You are now ready to provide access to your APIs by granting access tokens to the Auth0 API. By including specific scopes, you can control a client's application to some or all of the APIs represented by the Auth0 API.
+
+:::panel-info Authorization Flows
+
+The rest of this article covers use of the [Implicit Grant](/api-auth/grant/implicit) to reflect the sample. You can, however, use whichever flow best suits your needs.
+
+* If you have a **Non Interactive Client**, you can authorize it to request access tokens to your API by executing a [client credentials exchange](/api-auth/grant/client-credentials).
+* If you are building a **Native App**, you can implement the use of [Authorization Codes using PKCE](/api-auth/grant/authorization-code-pkce).
+:::
+
+The app initiates the flow and redirects the browser to Auth0 (specifically to the ``/authorize` endpoint), so the user can authenticate.
+
+```text
+https://YOUR_AUTH0_DOMAIN/authorize?
+scope=read:contacts%20read:calendar&
+audience=organize&
+response_type=id_token%20token&
+client_id=YOUR_CLIENT_ID&
+redirect_uri=http://localhost:3000&
+nonce=NONCE
+```
+The SPA executes this call whenever the user clicks **Login**.
+
+![SPA Home before Login](/media/articles/api-auth/tutorials/multiple-apis-one-resource-server/home.png)
+
+:::panel-info Request Parameters
+For additional information on the call's parameters, refer to the [docs on executing an implementing the Implicit Grant](/api-auth/grant/implicit#1-get-the-user-s-authorization).
+:::
+
+Lock handles the login process (you can log in with a user that exists in a [Connection](/identityproviders) enabled for your Auth0 Client).
+
+![SPA Login](/media/articles/api-auth/tutorials/multiple-apis-one-resource-server/lock.png)
+
+Next, Auth0 authenticates the user. If this is the first time the user goes through this flow, they will be asked to consent to the scopes that are given to the Client (such as post messages or list contacts).
+
+If the user consents, Auth0 continues the authentication process, and upon completion, redirects them back to the app with an `access_token` in the hash fragment of the URI. The app can now extract the tokens from the hash fragment. In a Single Page Application (SPA) this is be done using JavaScript (see the `getAccessToken` function located in `index.html` for the sample app). The app can then use the `access_token` to call the API on behalf of the user.
```

Comment:
`see the `getAccessToken` function located in `index.html` for the sample app` : add inline the script maybe?